apps/explorer/locales/de-DE/translation.json

```diff
@@ -20,7 +20,6 @@
   "Copy token ID": "Token ID kopieren",
   "Daily": "täglich",
   "Download CSV": "CSV herunterladen",
-  "Fiat price": "Fiat Preis",
```

Comment:
For future references: You only have to delete it from the English file. Our automation system will take care of deleting it from the other files.